package.json

```
@@ -13,6 +13,7 @@
     "files": [
         "backend/*.{js,md}",
         "backend/typings",
+        "cypress/**/*",
```

I just thought... Can we do it shorter?

```suggestion
        "cypress",
```

I think that would work equally well, as it does with `backend/typings` or `dist` or any other directory below

---

Of course, it was just a leftover from a previous version of this line